app/models/miq_ae_yaml_export.rb

```diff
@@ -201,6 +201,8 @@ def get_method_filename(method_obj)
       "#{method_obj.name}#{FILE_EXTENSIONS[method_obj.language]}"
     elsif method_obj['location'] == 'uri'
       "#{method_obj.name}.uri"
+    elsif method_obj['location'] == 'expression'
+      "#{method_obj.name}.expr"
```

Comment:
@gmcculloug We use the YAML extension to store method properties file, and we fetch all the methods based on the extension.